build.gradle

```diff
@@ -48,10 +48,10 @@ subprojects {
 
     apply plugin: "com.diffplug.gradle.spotless"
 
-    def checkSubTask = project.tasks.findByName('check')
-    if(checkSubTask) {
-        tasks['check'].dependsOn checkSubTask
-    }
+//    def checkSubTask = project.tasks.findByName('check')
+//    if(checkSubTask) {
+//        tasks['check'].dependsOn checkSubTask
+//    }
```

Comment:
I need to disable this because it is circular dependencies on henson-plugin:check task (detected on latest spotless version). What does checkSubtask doing in this case ? do you have any concern if we remove this ?